Directory.Build.props

```diff
@@ -19,7 +19,8 @@
   <!-- Define vNext UAP Moniker -->
   <PropertyGroup>
     <UAPvNextVersion>10.0.16300</UAPvNextVersion>
-    <UAPvNextTFMFull>UAP,Version=v$(UAPvNextVersion)</UAPvNextTFMFull>
+    <UAPvNextTFI>UAP</UAPvNextTFI>
```

Comment:
Why did we have to do this? do we expect the Moniker name to change?

Reply:
I just felt like it make sense to extract it and avoid the duplication. I don't know if it will ever change...